packages/react/src/Button/ButtonBase.module.css

```diff
@@ -430,8 +430,6 @@
     &:disabled,
     &[aria-disabled='true']:not([data-loading='true']) {
       color: var(--button-invisible-fgColor-disabled);
-      background-color: var(--button-invisible-bgColor-disabled);
-      border-color: var(--button-invisible-borderColor-disabled);
```

Comment:
This change makes these tokens obsolete. Should we delete them entirely? cc @langermank

Reply:
Just took a few min to make a PR for this which we can test and just use this branch to bump versions if you want. 

Could we make a Storybook story just showing all button variants `disabled` in a row, move it to the `dev` stories and add a VRT test for it? 🙌